src/components/timeout-dialog/timeout-dialog.yaml

```diff
@@ -48,6 +48,10 @@ examples:
     keepAliveButtonText: "keepAliveButtonText"
     signOutButtonText: "signOutButtonText"
     synchroniseTabs: false
+    hideSignOutButton: false
+#   hideSignOutButton is experimental, please don't use in HMRC. without testing. Added at request of non-HMRC service,
+#   where a service only has an anonymous session, so no explicit log in action has been taken, and you might want to
+#   have no early sign out link rather along with the than a link saying something like [abandon attempt]
```

Comment:
sorry, reread and my suggestion before was bit verbose 😅 what about this the #316 bit should link to the issue 🤞 
```suggestion
    hideSignOutButton: false
#   hideSignOutButton is experimental, please don't use in HMRC services without testing with your users #316 
```